src/lib/atoms/Export/exportHelper.ts

```diff
@@ -0,0 +1,241 @@
+import React from 'react';
+import jsPDF from 'jspdf';
+import * as ImageExporter from 'html-to-image';
+import ExcelJS from 'exceljs';
+import { saveAs } from 'file-saver';
+import autoTable, { ColumnInput } from 'jspdf-autotable';
+//TODO: change file location after tests
+
+export const exportToTablePdf = (
+    data: Record<string, string>[],
+    columns?: Record<string, unknown>[],
+    fileName: string = 'document'
+) => {
+    try {
+        const doc = new jsPDF({
+            format: 'letter'
+        });
+
+        const pdfColumns = columns
+            ? columns.map((column) => ({ header: column.header, dataKey: column.key }))
+            : data
+                  .reduce((aggr: string[], val) => {
+                      aggr = [...aggr, ...Object.keys(val)];
+                      return aggr;
+                  }, [])
+                  .map((el) => ({ header: el, dataKey: el }));
+
+        autoTable(doc, {
+            columns: [...new Set(pdfColumns)] as ColumnInput[],
+            body: data
+        });
+
+        doc.save(`${fileName}.pdf`);
+    } catch (error) {
+        return error;
+    }
+};
+
+export const pdf = (HTMLelement: HTMLElement, fileName: string = 'document') => {
+    try {
+        const doc = new jsPDF({
+            format: 'a4',
+            unit: 'px'
+        });
+
+        doc.html(HTMLelement, {
+            callback(doc) {
+                doc.save(fileName);
+            },
+            x: 10,
+            y: 10
+        });
+    } catch (error) {
+        return error;
+    }
+};
+
+export type ImageFormats = 'toPng' | 'toJpeg';
+export const exportImage = async (
+    HTMLelement: HTMLElement,
+    format: ImageFormats = 'toPng',
+    fileName: string = 'document'
```

Comment:
Why `format` and `fileName` is required ?
This props can have default values (as they have) and by default use that values.